baselines/webworker.generated.d.ts

```diff
@@ -1143,9 +1234,9 @@ interface Request extends Object, Body {
     readonly redirect: RequestRedirect;
     readonly referrer: string;
     readonly referrerPolicy: ReferrerPolicy;
+    readonly signal: any;
```

Comment:
This went from a strongly typed `AbortSignal` to `any`; should it be at least `object`? `any` feels unsafe.

Reply:
I can change that.